src/client/app/header/app.header.controller.js

```diff
@@ -0,0 +1,27 @@
+﻿(function () {
+    'use strict';
+
+    angular
+        .module('app.header')
+        .controller('HeaderController', HeaderController);
+
+    HeaderController.$inject = ['$sessionStorage', '$rootScope', '$state'];
+
+    function HeaderController($sessionStorage, $rootScope, $state) {
+        var vm = this;
+
+        vm.loggedIn = !!$sessionStorage.token || false; // this value is impacting view, if user is logged in some
+        // elements from nav menu will be displayed
+        $rootScope.$on('$stateChangeStart',
+            function(){
+                if($sessionStorage.token){
+                    return vm.loggedIn = true;
+                }
+            });
+        vm.logOut = function(){
+            $sessionStorage.$reset();
+            $state.go($state.current, {}, {reload: true});
+            vm.loggedIn = false;
+        }
```

Comment:
Token is not a persistent object, security generates new token once per request to /api/auth and it has ttl for 10 days. After logout reseting your session storage is enough, but it's not very good that cleaned token still will be valid for remaining period of time. I've already created bug for this issue https://jira.softserveinc.com/browse/CHSP-225